pkg/xray/provider.go

```diff
@@ -102,8 +109,13 @@ func providerConfigure(d *schema.ResourceData, terraformVersion string) (interfa
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
There is an outstanding usage report ticket in RT @alexhung please let daniel know so there is a consistent approach

Reply:
we will address in the next PR, after it's done for Artifactory